recipe/meta.yaml

```diff
@@ -8,22 +8,28 @@ package:
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/streamlit_extras-{{ version }}.tar.gz
   sha256: 52a8cb3a4394735ee873c92c95fabf56d84cc2400ed0617bbc8b8259c0560276
+  patches:
+    - patches/0001-remove-the-usage-of-entrypoints.patch
 
 build:
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 0
+  number: 1
   # s390x is missing streamlit
   skip: true  # [py<38 or s390x]
 
+
 requirements:
+  build:
+    - patch  # [not win]
+    - m2-patch  # [win]
   host:
     - python
     - poetry-core
     - pip
     - setuptools
     - wheel
   run:
-    - python <3.9.7|>3.9.7,<4.0
+    - python <3.9.7,>3.9.7,<4.0
```

Comment:
python != 3.9.7, would that work ?

Reply:
Will probably work. Just wanted to keep the amount of changes to a minimum. | might have been a typo.